src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/VariantsSparkSinkUnitTest.java

```diff
@@ -64,20 +67,20 @@ private void shutdownMiniCluster() {
         };
     }
 
-    @Test(dataProvider = "loadVariants", groups = "spark")
+    //@Test(dataProvider = "loadVariants", groups = "spark")
```

Comment:
@tomwhite Why did you disabled these tests?

Reply:
Oops. While debugging and I forgot to change it back. Fixed.